### DISCUSSION TITLE
data information of the mesh file of example3

### DISCUSSION THREAD

**Discussion by qianzhang1994** [alan-turing-institute/PeriPy#121] [Q&A]
Please check the data structure of the mesh file.  
https://github.com/alan-turing-institute/PeriPy/blob/feature/example3/peripy/examples/example3/175beam3620.msh
![image](https://user-images.githubusercontent.com/68472501/138764669-d05ab260-b40f-4cd9-98d6-055a1820847b.png)
![image](https://user-images.githubusercontent.com/68472501/138764711-ef7bd6b1-5ead-49de-b1c9-da7a14e7dc85.png)
There is no  'tetra'.

**Comment by mark-hobbs** [comment]
Hi Qian Zhang, which branch are you on? This example should work if you are on the following branch - feature/example3. This branch will be merged with the develop and master branch in the near future. The code has been updated to accept input files that only define the nodal coordinates (meshless). Note that the original code required a topological mesh.

**Reply by qianzhang1994** [reply]
Okay, Thanks. I only copy the example3 to original code. I think you have updated other codes, such as model.py. I will study the new model.
